scripts/release.ps1

```diff
@@ -1,5 +1,35 @@
+#Requires -Version 5.0
+
+param (
+    [parameter(Mandatory = $false)] [string]$PushImageToLibrary = "rancher"
+)
+
 $ErrorActionPreference = 'Stop'
 
 Import-Module -WarningAction Ignore -Name "$PSScriptRoot\utils.psm1"
 
 Invoke-Script -File "$PSScriptRoot\ci.ps1"
+
+& "$PSScriptRoot\version.ps1" | Out-Null
+
+$baseTag = "wmi-exporter-package:$($env:VERSION)"
+$currentTag = "rancher/$baseTag"
+$pushTag = "$PushImageToLibrary/$baseTag"
+
+$currentReleaseId = (docker images $currentTag --format "{{.ID}}")
+$pushedReleaseId = (docker images $pushTag --format "{{.ID}}")
+if ($currentReleaseId -ne $pushedReleaseId) {
+    docker tag $pushTag "$pushTag-bak" | Out-Null
+    docker tag $currentTag $pushTag | Out-Null
+}
+
+docker push $pushTag
+if ($?) {
+    docker rmi "$pushTag-bak" | Out-Null
+    docker rmi $currentTag | Out-Null
+    Write-Host "$pushTag was PUSHED"
+} else {
+    docker tag "$pushTag-bak" $pushTag | Out-Null
+    docker rmi "$pushTag-bak" | Out-Null
```

Comment:
What are the `bak` images for?

Reply:
copied from https://github.com/rancher/wmi_exporter-package/blob/8e0c309b986cfbfb96ec1dba0a40df612f149f36/scripts/windows/release.ps1#L18, but seems like they just create backup images in case the docker push fails (e.g. if I'm trying to push a local copy to the `rancher/` registry instead of `arvindiyengar/` by accident)